src/pkg/util/fs/helper_test.go

```diff
@@ -0,0 +1,39 @@
+package fs
+
+import "testing"
+
+func TestIsFile(t *testing.T) {
+	if IsFile("/etc/passwd") != true {
+		t.Errorf("IsFile returns false for file")
+	}
+}
+
+func TestIsDir(t *testing.T) {
+	if IsDir("/etc") != true {
+		t.Errorf("IsDir returns false for directory")
+	}
+}
+
+func TestIsLink(t *testing.T) {
+	if IsLink("/proc/mounts") != true {
```

Comment:
If we are going to build a client (to run remote builds) from a non-linux machine e.g. a mac then we can't expect all of these test targets to exist.

Reply:
@dctrud Good point ! Or simply rename it to `helper_linux_test.go` ?